src/Samples/DIPS.Xamarin.UI.Samples/Controls/Sheet/SheetPage.xaml.cs

```diff
@@ -39,12 +39,27 @@ public class SheetPageViewModel : INotifyPropertyChanged
         private bool m_shouldRememberPosition;
         private string m_contentColor;
         private string m_headerColor;
+        private bool m_cancelCanExecute = true;
+        private string m_title;
+        private bool m_hasActionButton;
 
         public SheetPageViewModel()
         {
             OpenSheetCommand = new Command(() => IsSheetOpen = true);
             OnOpenCommand = new Command<string>(SheetOpened);
             OnCloseCommand = new Command<string>(SheetClosed);
+            CancelCommand = new CancelSheetCommand<string>(
+                str =>
+                {
+                    
+                },
+                str => m_cancelCanExecute, str => true);

```

Comment:
Open the `canCloseSheet` lamba and add `//Do logic to determine if the sheet should close` and return `true`